packages/app/src/app/overmind/namespaces/dashboard/types.ts

```diff
@@ -41,3 +44,5 @@ export enum sandboxesTypes {
   ALWAYS_ON = 'ALWAYS_ON',
   DISCOVER = 'DISCOVER',
 }
+
+export type RepositoryBranch = RecentlyAccessedBranchesQuery['me']['recentBranches'][number];
```

Comment:
is this the right way to do it if there's no fragment generated for a branch type?